src/psyclone/transformations.py

```diff
@@ -2262,3 +2262,82 @@ def apply(self, sched):
         schedule.addchild(data_dir, index=0)
 
         return schedule, keep
+
+
+class ACCRoutineTrans(Transformation):
+    '''
+    Transform a kernel subroutine through the addition of a "!$acc routine"
```

Comment:
Change "through the addition of" to "by adding"

Reply:
Done.